patches/1.1.0-grasshopper/oids.diff

```diff
@@ -0,0 +1,22 @@
+diff --git a/crypto/objects/objects.txt b/crypto/objects/objects.txt
+index 645591b..b8e2467 100644
+--- a/crypto/objects/objects.txt
++++ b/crypto/objects/objects.txt
+@@ -1297,12 +1297,12 @@ member-body 643 100 111 : subjectSignTool       : Signing Tool of Subject
+ member-body 643 100 112        : issuerSignTool        : Signing Tool of Issuer
+
+ #GOST R34.13-2015 Grasshopper "Kuznechik"
+-                       : grasshopper-ecb
+-                       : grasshopper-ctr
+-                       : grasshopper-ofb
+-                       : grasshopper-cbc
+-                       : grasshopper-cfb
+-                       : grasshopper-mac
++cryptopro 37 1 : GRASSHOPPER-ECB : grasshopper-ecb
++cryptopro 37 2 : GRASSHOPPER-CTR : grasshopper-ctr
++cryptopro 37 3 : GRASSHOPPER-OFB : grasshopper-ofb
++cryptopro 37 4 : GRASSHOPPER-CBC : grasshopper-cbc
++cryptopro 37 5 : GRASSHOPPER-CFB : grasshopper-cfb
++#grasshopper-mac
+
+ # Definitions for Camellia cipher - CBC MODE
```

Comment:
Instead of having people patch OpenSSL, you might want to consider either giving them an extra config file with the OIDs (see `perldoc doc/apps/config.pod` in the OpenSSL source) or adding the dynamically when the engine is loaded, using `OBJ_create()` (see `perldoc doc/crypto/OBJ_nid2obj.pod` in the OpenSSL source).